### PR TITLE
Upgrade to 1.0.0-rc1

### LINKED ISSUE #57: upgrade to 1.0.0-beta.3
Hi there,
By any chance, are you looking into upgrading the app to latest version ? 
I'm trying to help Framasoft test the beta...

Thanks for any answer (even negative, I would understand).

### PR DESCRIPTION
**Pull request by yalh76**
## Problem
- *Fixes #57*

## Solution
- *Upgrade to 1.0.0-beta.3*

## Wating for:
- [x] https://framagit.org/framasoft/mobilizon/-/merge_requests/525 to add `--listen-port` option
- [x] https://framagit.org/framasoft/mobilizon/-/issues/368 to solve `--instance-name`

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/mobilizon_ynh%20PR58%20(yalh76)/)

### CODE REVIEW

**Comment by Gredin67** [comment]
Hi, I tested installing this PR on a x86 and it failed. What is the status of this app?
Here is the log of the failed installed https://paste.yunohost.org/raw/adifofabuv

**Comment by yalh76** [comment]
> Hi, I tested installing this PR on a x86 and it failed. What is the status of this app?
> 
> Here is the log of the failed installed https://paste.yunohost.org/raw/adifofabuv

This PR is not ready, I'm waiting they provide a new beta or have to work on an intermediary version

**Comment by Gredin67** [comment]
Obviously they are going to publish the first release within the next days or weeks. I'll let you know if I get some more info.

**Comment by Gredin67** [comment]
https://paste.yunohost.org/raw/iticafujez

**Comment by alexAubin** [comment]
(Sounds like it's just complaining about missing cmake ?

```
/bin/sh: 1: cmake: not found
```
)

**Comment by Gredin67** [comment]
adding `cmake` in the dependencies of the package solved the problem. I am wondering if `build-essential` is necessary, as you already define `make` and `gcc`.

File: conf/nginx.conf [hunk 1]
```diff
@@ -4,16 +4,30 @@ location / {
...
   proxy_set_header Host $http_host;
 
+  proxy_redirect off;
+
   proxy_pass http://localhost:__PORT__;
 
   client_max_body_size 16m;
 
+  location ~* \.(css|js)$ {
+    root /home/mobilizon/live/priv/static;
```
Comment (alexAubin) [inline conf/nginx.conf:24]:
```suggestion
    root __FINALPATH__/mobilizon/priv/static;
```

(not 100% sure about the exact syntax but had to do this to fix the blank page (no css/js loading) ... my exact path was : /var/www/mobilizon/mobilizon/priv/static  (note the two /mobilizon !))

**Comment by alexAubin** [comment]
Yolomerging to testing to move forward

**Pull request merged**